content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -20,257 +17,259 @@ typically requires a VPC or a VPN, as well as a container network that
 
-*   Generate DNS configuration file to be used in the VMs. This will allow apps on the VM to resolve
-cluster service names, which will be intercepted by the sidecar and forwarded.
+5.    If the VM only calls services in the mesh, you can skip this step. Otherwise, add the ports the VM exposes to
+    the cluster.env file with the following command. You can change the ports later.
 
     {{< text bash >}}
-    $ @install/tools/setupMeshEx.sh@ generateDnsmasq
+
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> cluster.env
+
     {{< /text >}}
 
-    Here's an example generated file
+6.    Extract the initial keys for the service account to use on the VMs.
+
+    {{<text bash>}}
+
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+        -o jsonpath='{.data.root-cert\.pem}' |base64 --decode > root-cert.pem
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+        -o jsonpath='{.data.key\.pem}' |base64 --decode > key.pem
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+          -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
 
-    {{< text bash >}}
-    $ cat kubedns
-    server=/svc.cluster.local/10.150.0.7
-    address=/istio-mixer/10.150.0.8
-    address=/istio-pilot/10.150.0.6
-    address=/istio-citadel/10.150.0.9
-    address=/istio-mixer.istio-system/10.150.0.8
-    address=/istio-pilot.istio-system/10.150.0.6
-    address=/istio-citadel.istio-system/10.150.0.9
     {{< /text >}}
 
 ### Setting up the machines
 
-As an example, you can use the following "all inclusive" script to copy
-and install the setup:
+1.    Install the debian package:
+
+    {{< text bash >}}
+
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ dpkg -i istio-sidecar.deb
 
-{{< text bash >}}
-$ export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
-$ export SERVICE_NAMESPACE=vm
-{{< /text >}}
+    {{< /text >}}
 
-If you are running on a GCE VM, run
+2.    Copy the cluster.env and the *.pem files to the VM.
 
-{{< text bash >}}
-$ @install/tools/setupMeshEx.sh@ gceMachineSetup VM_NAME
-{{< /text >}}
+3.    Add the IP address of the Istio gateway to /etc/hosts or to the DNS server. The following is and example
+    of /etc/hosts:
 
-Otherwise, run
+    {{< text bash>}}
 
-{{< text bash >}}
-$ @install/tools/setupMeshEx.sh@ machineSetup VM_NAME
-{{< /text >}}
+    $ echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
 
-GCE provides better user experience since node agent can always relies on
-GCE metadata instance document to authenticate to Citadel. For everything
-else, e.g., on-prem or raw VM, we have to bootstrap a key/cert as credential,
-which typically has a limited lifetime. And when the cert expires, you have to
-rerun the above command.
+    {{< /text >}}
 
-Or the equivalent manual steps:
+4.    Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs`.
```

Comment:
ordered lists in markdown all begin with a 1.  (i.e.) it is not necessary to specify the value 4, the markdown parser should sort that out.  Also removal of the spaces will make for a better rendering in the lead up to  Install.